.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This file (and the two `.pyc` files below) should *not* be tracked by git! They are specific to your local set-up (which is probably different to everyone elses!)...[See `gitignore`](https://git-scm.com/docs/gitignore)!